src/polyswarmtransaction/transaction.py

```diff
@@ -59,6 +59,10 @@ def __message(self, public_key: PublicKey) -> str:
 
     @staticmethod
     def load_key(private_key: HexBytes) -> PrivateKey:
```

Comment:
Change the type here to match

Reply:
Yes but will get big: `Union[HexBytes,PrivateKey]`

Confirm?

Reply:
Done on 727a7ed1f9fb879c1bfb0302c1aae46645b2aa99